stable/mercure/Chart.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: v1
+appVersion: "0.3.3"
+description: The Mercure hub allows to push data updates using the Mercure protocol to web browsers and other HTTP clients in a convenient, fast, reliable and battery-efficient way
+name: mercure
+version: 0.1.0
```

lets set to 1.0.0 since it is in stable
```suggestion
version: 1.0.0
```